app/views/criteria/ineligible.html.erb

```diff
@@ -1 +1,34 @@
-Ineligible Criteria page
\ No newline at end of file
+<div class="ineligible">
+  <span class="ineligible__icon"></span>
+  <div class="container">
```

Comment:
The main colour here in the design is #343434 - I've created this in the dashboard branch as `$grey-xxdark` but happy to stick with the already defined #242424 colour variable if preferred

Reply:
Yeah I know, I was oing to leave it as is to avoid "50 shades of grey", if that's ok.